packages/fxa-auth-server/lib/backendService.js

```diff
@@ -173,7 +173,7 @@ module.exports = function createBackendServiceAPI(log, config, serviceName, meth
       // Next are query params as a dict, if any.
       const query = validation.query ? await validate('query', args[i++], querySchema) : {};
       // Next is request payload as a dict, if any.
-      const payload = validation.payload ? await validate('request', args[i++], payloadSchema) : {};
+      const payload = validation.payload ? await validate('request', args[i++], payloadSchema) : null;
```

Comment:
If the problem is only with GET/PUT requests, should we make null the default for just these two verbs to avoid falling afoul of expectations elsewhere? We have opts.method we could use.